### PR TITLE
T1329

### PR DESCRIPTION
**Pull request by k88hudson**
context-switcher plugin

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/decfabc876a94f3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/decfabc876a94f3/output.txt

Total script time: 0.16 mins

File: templates/supported/shared/plugins/shrink/popcorn.shrink.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// PLUGIN: Context Switcher
```
Comment (humphd) [inline templates/supported/shared/plugins/shrink/popcorn.shrink.js]: Change references to "Context Switcher" in this file to shrink.

File: templates/supported/shared/plugins/shrink/popcorn.shrink.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+(function ( Popcorn ) {
+
+  /**
+   * Context Switcher Popcorn plug-in
```
Comment (humphd) [inline templates/supported/shared/plugins/shrink/popcorn.shrink.js]: Here too.

File: templates/supported/shared/plugins/shrink/popcorn.shrink.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+   *
+   */
+
+  Popcorn.plugin( "shrink", {
+
+    manifest: {
+      about: {
+        name: "Popcorn Context Switcher Plugin",
```
Comment (humphd) [inline templates/supported/shared/plugins/shrink/popcorn.shrink.js]: Here.

**Comment by jbuck** [comment]
Rebased by myself: https://github.com/mozilla/butter/compare/c7c27a1...a098aa1

**Pull request closed without merging**